scala-xml/src/main/scala/org/http4s/scalaxml/ElemInstances.scala

```diff
@@ -46,7 +46,7 @@ trait ElemInstances {
   implicit def xmlEvents[F[_]](implicit F: Concurrent[F]): EntityDecoder[F, Stream[F, XmlEvent]] =
     EntityDecoder.decodeBy(MediaType.text.xml, MediaType.text.html, MediaType.application.xml) {
       msg =>
-        DecodeResult.successT(msg.bodyText.through(fs2.data.xml.events()))
+        DecodeResult.successT(msg.bodyText.through(fs2.data.xml.events(includeComments = true)))
```

Comment:
This is necessary to make the new round-trip test pass. But is it actually what we want?

Reply:
I conflated normalization in the generator with normalization in the test in a way I probably shouldn't have.  I guess the question is whether comments are part of normalized output.  I think it belongs, perhaps with an option to turn it off.